airbyte-integrations/connectors/source-genesys/source_genesys/spec.json

```diff
@@ -0,0 +1,51 @@
+{
+  "documentationUrl": "https://docs.airbyte.io/integrations/sources/genesys",
+  "connectionSpecification": {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "Genesys Connector Configuration",
+    "type": "object",
+    "required": ["start_date", "tenant_endpoint", "client_id", "client_secret"],
```

Comment:
According to the documentation, you said

> You can follow the documentation on [API credentials](https://developer.genesys.cloud/authorization/platform-auth/use-client-credentials#obtain-an-access-token)

Do you plan to add the spec for the token and support the TokenAuthenticator?

Reply:
@YiyangLi not sure i follow, for now I’m only using OAuth to keep things simple. Tokens can be used directly but they expire, so using pattern i borrowed from other live connectors which use similar OAuth approach.

Reply:
Can you update the doc since only OAuth2 is supported?

Reply:
@YiyangLi updated docs